srv/relayer.go

```diff
@@ -120,15 +122,39 @@ func (r *Relayer) SendUserOperation() modules.BatchHandlerFunc {
 			attribute.Int64("wait_timeout", int64(r.waitTimeout)),
 		)
 
+		var computeOPHashes = func(err error, hash common.Hash) {
```

Comment:
Please add a func documentation and a few comments for the context and what we're accomplishing here.